games-board/netmaumau/netmaumau-0.16-r1.ebuild

```diff
@@ -32,6 +32,10 @@ DEPEND="
 
 S=${WORKDIR}/${P}-client
 
+src_prepare() {
+	epatch "${FILESDIR}/connect-to-old-servers.patch"
```

Comment:
Since the PR-commit is the last commit in your branch, it's extremely easy to fix it.

Just change the ebuild, start ```git gui```, click on "Amend Last Commit" in the lower right corner, stage the changed files and commit and force-push.